lib/matplotlib/backends/backend_pgf.py

```diff
@@ -582,11 +582,10 @@ def _print_pgf_path_styles(self, gc, rgbFace):
         if dash_list is None:
             writeln(self.fh, r"\pgfsetdash{}{0pt}")
         else:
-            dash_str = r"\pgfsetdash{"
-            for dash in dash_list:
-                dash_str += r"{%fpt}" % dash
-            dash_str += r"}{%fpt}" % dash_offset
-            writeln(self.fh, dash_str)
+            writeln(self.fh,
+                    r"\pgfsetdash{"
+                    + "".join(r"{%fpt}" % dash for dash in dash_list)
+                    + "r}{%fpt}" % dash_offset)
```

Comment:
The r was a raw string qualifier before and seems to have slipped into the string.

Reply:
cf @tacaswell's comment about none of this being tested :p 
fixed

Reply:
On the danger of being pedantic, I would simply write
~~~
            dash_pattern = "".join(r"{%fpt}" % dash for dash in dash_list)
            writeln(self.fh,
                    r"\pgfsetdash{%s}{%fpt}" % (dash_pattern, dash_offset))
~~~